requirements/common.txt

```diff
@@ -92,6 +99,10 @@ psycopg2==2.6.2
 
 pyasn1==0.1.9
 pyasn1-modules==0.0.8
+
+# Python wrapper around OpenSSL
+# Needed by ndg-httpsclient
+pyOpenSSL==16.0.0
```

Comment:
these versions probably need updating, but also we already have some of them in `requirements/scrapy.txt`; so some cleanup needed.